js/src/widget_ngl.js

```diff
@@ -15,7 +15,7 @@ require('jquery-ui/ui/widgets/draggable');
 require('jquery-ui/ui/widgets/slider');
 require('jquery-ui/ui/widgets/dialog');
 require('jquery-ui/themes/base/all.css');
-require('./css/dark.css');
+// require('./css/dark.css'); // How to switch theme?
```

Comment:
@arose the css content will be bundled in `nglview-js-widgets`, any idea how to switch the theme? 
(In NGL example, it specify the file path, which is not applicable in this case (may be I am wrong?)).